tools/htseq.wdl

```diff
@@ -9,24 +9,34 @@ task count {
     input {
         File bam
         File gtf
-        String provided_strandedness
-        String inferred_strandedness = ""
         String outfile = basename(bam, ".bam") + ".feature-counts.txt"
-        Int added_memory_gb = 20
+        String provided_strandedness = ""
```

Comment:
Is this behavior correct? If both are omitted, then they'll both be `""`. Did I correctly read the below to see that the strandedness parameter will end up as `unknown-strand` at runtime and will cause a runtime error in HTSeq? Ideally, it would be a pre-run check that at least one was set. I don't know of a way to express that in WDL. I think CWL has the ability to specify this (though it may not be straightforward).

Reply:
It's checked as part of `parse_input` at the start of the workflow. However if you import just this task there isn't an elegant catch for the built-in.